src/GetOpt.php

```diff
@@ -502,7 +502,7 @@ public function offsetExists($offset): bool
      * @param mixed $offset
      * @return mixed
      */
-    public function offsetGet($offset)
+    public function offsetGet($offset): mixed
```

Comment:
mixed is not available in php 7 - is that a warning in php 8 now?

Reply:
we have to go with that backward compatible comment
https://php.watch/versions/8.1/ReturnTypeWillChange